save-cloud-common/src/commonMain/kotlin/org/cqfn/save/domain/Role.kt

```diff
@@ -0,0 +1,32 @@
+package org.cqfn.save.domain
+
+/**
+ * User roles
+ */
+enum class Role {
```

Comment:
may be split this class into two roles? one for SAVE globally and second is for projects 

This is **NOT** required

Reply:
Now we have `user_id` and `admin_ids` in `Project`, which is more or less a per-project role. Since there is no support for per-project roles in spring-security, I think we don't need it as a separate entity yet.